RecoHI/HiEvtPlaneAlgos/interface/HiEvtPlaneFlatten.h

```diff
@@ -147,7 +142,7 @@ class HiEvtPlaneFlatten {
     caloCentRefVal_/=refmax-refmin+1.;
     if(caloCentRefMinBin_<0) return 1.;
     int indx = getOffsetIndx(centbin,vtx);
-    if(caloCentRefVal_==0 || getPtDB(indx)==0) return 1.;
+    if(indx < 0 or caloCentRefVal_ == 0 or getPtDB(indx)==0) return 1.;
```

Comment:
Could I advice against such style? `||` -> `or` The style should be kept consistent, i.e., `||`